addons/inkgd/editor/ink_dock.tscn

```diff
@@ -1,7 +1,22 @@
-[gd_scene load_steps=3 format=2]
+[gd_scene load_steps=4 format=2]
```

Comment:
Can you walk me through the changes in this file (mostly margin stuff it appears). Is it editor noise or was something touched on purpose?

Reply:
I'm resolving this, since it was dealt with in another comment.